### ISSUE TITLE
Move Issues link below all the displayed "Dashboards" for users with multiple roles

### ISSUE THREAD

**Issue by blesildaramirez** [pkp/pkp-lib#11070]
## Issue Description
When a user has multiple roles, more than one "Dashboards" menu is displayed. (see screenshot below for example)
The Issues link should be below all of the submission dashboards available for the user.

![Image](https://github.com/user-attachments/assets/0c912ccd-a6f6-4036-b87c-c1ad9e38dfaa)

## How to reproduce / Current behavior
Login as a user with reviewer, author, and journal editor roles. Multiple submission dashboards should appear on the sidemenu. 

## What application are you using?
 OJS 3.5.0

## Testing method
- Manual

## Action
Move the Issues link below the last Submissions dashboard link: (e.g from the screenshot: "My Submissions")

---
Pull requests:
**OJS**
stable-3_5_0: https://github.com/pkp/ojs/pull/4733
main: https://github.com/pkp/ojs/pull/4734

**Comment by asmecher** [comment]
Added a review [here](https://github.com/pkp/ojs/pull/4734#pullrequestreview-2679178390), thanks!

**Comment by asmecher** [comment]
I approved both PRs, thanks, @blesildaramirez!
